processor/src/main/java/org/mapstruct/ap/internal/model/PropertyMapping.java

```diff
@@ -563,6 +563,20 @@ private String getSourcePresenceCheckerRef( SourceReference sourceReference ) {
                 if ( propertyEntry.getPresenceChecker() != null ) {
                     sourcePresenceChecker = sourceParam.getName()
                         + "." + propertyEntry.getPresenceChecker().getSimpleName() + "()";
+
```

Comment:
I think it would be good to extract this code and put it into its own method. Something like: `constructSourcePresenceCheck` don't you think?

Reply:
I saw its context now.. Its already a small method. So.. lets leave it like you propose.